### PR TITLE
KK-34 | Styles

### PR DESCRIPTION
**Pull request by midori0507**
- Add alias for easier sass var import.
- Add normalizer css, inject Helsinki fonts.
- Extract basic style from opencity.design.

Extra: Join CI jobs to reduce machine booting time -> faster CI

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+SASS_PATH=./src/assets/styles
```
Comment (Nizar-Rahme) [inline .env]: `.env` file shouldn't be pushed to GitHub

File: src/domain/app/header/Header.module.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+.appName {
+  margin-left: 30px;
+}
+
+.logo {
+  @include helsinki-logo($hel-black);
+  background-position: center;
+  width: $logo-width;
+  height: inherit;
+}
```
Comment (Nizar-Rahme) [inline src/domain/app/header/Header.module.scss]: Wouldn't it be better to stick to the naming convention with other projects, which is starting `scss` files with small letters?

File: .env (hunk 1 above)
Comment (jimppa) [reply .env]: Actually it should if following CRA: https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env

Comment (midori0507) [reply .env]: They should, according to create-react-app docs :D https://create-react-app.dev/docs/adding-custom-environment-variables#docsNav

Comment (Nizar-Rahme) [reply .env]: Fair enough

File: src/domain/app/header/Header.module.scss (hunk 2 above)
Comment (midori0507) [reply src/domain/app/header/Header.module.scss]: Done

**Pull request merged**